### PR TITLE
Modify salesforce_ftp_integration module.md file

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Modify salesforce_ftp_integration module.md file

### CODE REVIEW

File: docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md [hunk 1]
```diff
@@ -1,82 +1,34 @@
-Guide on Integrating Salesforce with FTP.
+Template for Integrating Salesforce with FTP
 
-# Guide Overview
+# Integrating Salesforce with FTP 
 
-## About
+This is a template for [Import Contacts into Salesforce Using FTP](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/1/). Please refer to it for more details on what you are going to build here. This template provides a starting point for your scenario.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md]:
```suggestion
This is a template for the [Import Contacts into Salesforce Using FTP tutorial](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/1/). Please refer to it for more details on what you are going to build here. This template provides a starting point for your scenario. 
```

File: docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md [hunk 2]
```diff
@@ -108,17 +60,7 @@ FTP_PATH="<Path in the FTP server you added conatats.csv (eg: /home/ftp-user/sfd
...
-Open the project with VS Code. The integration implementation is written in the `src/salesforce_ftp_integration/main.bal` file.
-
-Here the `ftpServerConnector` service is running on `remoteServer`, which listens to the configured FTP server
-location. When a CSV file is added to the FTP server, the file content will be retrieved and inserted into the
-Salesforce using `sfBulkClient`.
-
-## Testing
-
-First let’s build the module. Navigate to the project root directory and execute the following command.
+Let’s build the module. Navigate to the project root directory and execute the following command.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md]: Shouldn't we build before we test?

Comment (Nipunich) [reply docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md]: These configurations are required to build the module.

File: docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md (hunk 1 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md]: Fixed

File: docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md (hunk 2 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md]: These configurations are required to build the module.

**Pull request merged**